### PR TITLE
Add .NET Standard 2.0 target framework

### LINKED ISSUE #1449: Produce a .NET Standard 2.0 assembly
Because we can, because it's [recommended by MS folks](https://twitter.com/davidfowl/status/1025434242832945154), and because it has a much smaller dependency graph.

### PR DESCRIPTION
**Pull request by thomaslevesque**
Fixes #1449

### CODE REVIEW

File: src/FakeItEasy/FakeItEasy.csproj [hunk 1]
```diff
@@ -50,10 +50,20 @@
     <PackageReference Include="System.Runtime.Loader" Version="4.0.0" />
   </ItemGroup>
 
+  <!-- .NET Standard 2.0 -->
```
Comment (blairconrad) [inline src/FakeItEasy/FakeItEasy.csproj]: SUPER NIT. I'd've expected this section before .NET Standard 1.6, as we seemed to be putting modern frameworks on the top. This is not really a big deal. Carry on!

**Comment by thomaslevesque** [comment]
>     1. add an API approval in `tests/FakeItEasy.Tests.Approval/ApiApproval.cs`
>     2. PdbGit it in `tools/FakeItEasy.Build/Program.cs`
>     3. Update the `README.md`

Doh! Of course.

File: src/FakeItEasy/FakeItEasy.csproj (hunk 1 above)
Comment (thomaslevesque) [reply src/FakeItEasy/FakeItEasy.csproj]: Sure, will do

**Comment by thomaslevesque** [comment]
oops, don't know what I just did, but I messed up my branch...

**Comment by blairconrad** [comment]
Thanks, @thomaslevesque !

**Comment by thomaslevesque** [comment]
Thanks!

**Pull request merged**